src/main/resources/view/ProfileListCard.fxml

```diff
@@ -3,11 +3,11 @@
 <?import javafx.geometry.Insets?>
 <?import javafx.scene.control.Label?>
 <?import javafx.scene.layout.ColumnConstraints?>
-<?import javafx.scene.layout.FlowPane?>
 <?import javafx.scene.layout.GridPane?>
 <?import javafx.scene.layout.HBox?>
 <?import javafx.scene.layout.Region?>
 <?import javafx.scene.layout.VBox?>
+<?import javafx.scene.layout.FlowPane?>
```

Comment:
for line 17 (can't comment on it since the code has not been changed)
perhaps the minHeight could be decreased? due to the deletion of Address there is now extra padding around the contents. you could try around 85-90

Reply:
I agree with your suggestion. Will keep this in mind when we fine-tune the UI in the upcoming milestone.